app/javascript/components/widget-controller.jsx

```diff
@@ -73,6 +75,20 @@ export const WidgetController = ({ client, cityName }) => {
           return <DisconnectedDisplay cityName={cityName} error={error} />;
         }
 
+        if (
+          R.pathOr(
+            data.location.widgets.enabled === [],
+            ['NullPointerhandler', 'condition'],
+            {
+              NullPointerhandler: {
+                conditon: data.location.widgets.enabled === [],
+              },
+            },
+          )
```

Comment:
What's the advantage of using ramda in this conditional?

Reply:
In this case, ramda isn't much better than a lodash `get` since we're using it to just safely access a path in an object. The pay off with ramda is functional composition: https://github.com/ramda/ramda/wiki/Cookbook